fixity/reporting.py

```diff
@@ -1,16 +1,25 @@
 import calendar
 import json
+from datetime import datetime
+from typing import Optional
 
 import requests
 
+from .models import Report
 from .utils import check_valid_uuid
 
 
 class ReportServiceException(Exception):
     pass
 
 
-def post_pre_scan_report(aip, start_time, report_url, report_auth=(), session_id=None):
+def post_pre_scan_report(
+    aip: str,
+    start_time: datetime,
+    report_url: str,
+    report_auth: str = (),
```

Comment:
Why is this typed as `str` when it expects a `tuple`?

Reply:
Aah yes, I missed to see [this](https://github.com/artefactual/fixity/blob/6eb4807a802dacee36beda99526ebddc05847817/fixity/fixity.py#L92-L96).